fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterSystemStmtAnalyzer.java

```diff
@@ -50,5 +57,35 @@ public Void visitFrontendClause(FrontendClause frontendClause, ConnectContext co
             }
             return null;
         }
+
+        @Override
+        public Void visitModifyFrontendHostClause(ModifyFrontendAddressClause clause, ConnectContext context) {
```

Comment:
If you should reject the operation when the srcHost and the destHost are the same, it's non-sense.